instagram/src/components/PostContainer/PostHeader.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const PostContainer = (props) => {
```

Comment:
you named this component `PostContainer`, but I think you wanted to call it `PostHeader`

Reply:
🤔Apparently so.